### ISSUE TITLE
Tests for packfiles

### ISSUE THREAD

**Issue by axic** [clehner/abstract-pull-git-repo#1]
Can you add canonical tests for packfiles?

**Comment by clehner** [comment]
I agree that tests should be added for the packfile-related methods. However, I do not have immediate plans to add them. Are you looking to build a new pull-git-repo implementation?

**Comment by axic** [comment]
I did a PoC for using Ethereum+IPFS/Swarm as a backend: https://github.com/axic/git-remote-mango. Code quality is that of a hackathon 😃

**Comment by clehner** [comment]
Nice!
I would think that since IPFS does data deduplication, packfiles might not be necessary.

**Comment by axic** [comment]
For IPFS, conceptually, the objects are much better. Due to speed concerns and if the Git repo is only used for archival purposes (i.e. it is only pushed at every release, while the main development is on a different remote) using a packfile would be advantageous.

I plan to move on to Swarm, where packfiles wouldn't be needed at all.

I still think it would be good to have tests for the future :)
